packages/core/core/src/services/document-service/repository.ts

```diff
@@ -148,7 +148,12 @@ export const createContentTypeRepository: RepositoryFactoryMethod = (uid) => {
         assoc('documentId', createDocumentId()),
         // Merge new data into it
         (data) => merge(data, queryParams.data),
-        (data) => entries.create({ ...queryParams, data, status: 'draft' })
+        (data) =>
+          entries.create({
+            ...queryParams,
+            data,
+            status: hasDraftAndPublish ? 'draft' : 'published',
+          })
```

Comment:
is the status even necessary here? I believe entries.create totally ignores that attribute, so maybe we can just ignore it.

The clone will copy the publishedAt value of the original entry , and as all entries have that column (even if draft and publish is disabled) I believe it should work, wdyt?

Reply:
Hey, sorry for getting to this slowly, I see what you mean.

I think the problem was in `packages/core/content-manager/server/src/services/document-manager.ts` here:
[`b356982` (#20629)](https://github.com/strapi/strapi/pull/20629/commits/b35698277f72b0594d0eb7b4f4799edce11592c2)

With `publishedAt` always being set to null this line (150) was always merging publishedAt as null into data, regardless of D&P being enabled
https://github.com/strapi/strapi/blob/b35698277f72b0594d0eb7b4f4799edce11592c2/packages/core/core/src/services/document-service/repository.ts#L150-L151

With that removed we can rely on the publlishedAt retrieved from the above query:
https://github.com/strapi/strapi/blob/b35698277f72b0594d0eb7b4f4799edce11592c2/packages/core/core/src/services/document-service/repository.ts#L132-L141

Reply:
ah! 💡 that sounds like it